test/spec/modules/magniteAnalyticsAdapter_spec.js

```diff
@@ -1758,16 +1758,39 @@ describe('magnite analytics adapter', function () {
       });
     });
     describe('cookieless', () => {
-      beforeEach(() => {
-        magniteAdapter.enableAnalytics({
-          options: {
-            cookieles: undefined
-          }
-        });
-      })
+      // beforeEach(() => {
+      //   magniteAdapter.enableAnalytics({
+      //     options: {
+      //       cookieles: undefined
```

Comment:
it is on the comment now, but was spelling causing an issue?

Reply:
no it was something else (mot initializing the variable), this bit of code just does not even do anything so I commented out while testing and forgot to remove, I deleted now.